src/main/java/seedu/jarvis/model/financetracker/PaymentsOwedByUser.java

```diff
@@ -0,0 +1,63 @@
+package seedu.jarvis.model.financetracker;
+
+import java.util.ArrayList;
+
+/**
+ * Manages a list of debts owed by the user to others.
+ */
+public class PaymentsOwedByUser {
+    private ArrayList<PendingPayment> pendingPaymentsByUser;
+
+    /**
+     * Empty constructor to be used when there are no pending payments owed to the user.
+     */
+    public PaymentsOwedByUser() {
+        pendingPaymentsByUser = new ArrayList<PendingPayment>();
+    }
+
+    /**
+     * Default constructor to be used when JARVIS starts up.
+     */
+    public PaymentsOwedByUser(ArrayList<PendingPayment> paymentsOwed) {
+        if (paymentsOwed.size() == 0) {
+            //todo throw exception for no existing payments owed
+        } else {
+            this.pendingPaymentsByUser = paymentsOwed;
+        }
+    }
+
+    /**
+     * Adds a new pending payment that the user owes to another person.
+     * todo check arguments for this command (might need to change)
+     */
+    public void newDebtOwed(PendingPayment newPendingPayment) {
+        /*PendingPayment newPendingPayment = new PendingPayment(name, moneyAmount);
```

Comment:
Try not to leave commented out or outdated code in PR.

Reply:
alright noted with thanks!